api/exec/compose_stack.go

```diff
@@ -115,3 +123,129 @@ func createEnvFile(stack *portainer.Stack) (string, error) {
 
 	return "stack.env", nil
 }
+
+func createNetworkEnvFile(stack *portainer.Stack) error {
+	ss := NewStringSet()
```

Comment:
ss is for string set? why not give it a name that suits its purpose?

Reply:
Yes.  networkNameSet is okay?

Reply:
yes, or just `networkNames`?

Reply:
Sure

Reply:
Follow your another suggestion, it has already var named 'networkNames'. They are in one function. So it has to use another name.